shadow-cljs.edn

```diff
@@ -13,11 +13,28 @@
                           "process" false
                           "buffer" false
                           "crypto" false}}
-   :modules {:flureedb {:exports {default flureedb/flureedb}}}
+   :modules {:fluree-browser-sdk {:exports {default fluree-browser-sdk/fluree-browser-sdk}}}
    :build-hooks [(shadow.cljs.build-report/hook)]
    :devtools {:watch-dir "dev/browser"}}
 
-  :flureeworker
+  :fluree-node-sdk
+  {:target    :node-library
+   :output-to "out/nodejs/fluree-node-sdk.js"
+   :closure-defines {fluree.db.platform/BROWSER false}
+   :compiler-options {:optimizations :simple}
+   :exports   {:commit fluree-node-sdk/commit
+               :connect fluree-node-sdk/connect
+               :create fluree-node-sdk/create
+               :db fluree-node-sdk/db
+               :exists fluree-node-sdk/exists
+               :load fluree-node-sdk/load
+               :loadFromAddress fluree-node-sdk/loadFromAddress
+               :query fluree-node-sdk/query
+               :setLogging fluree-node-sdk/setLogging
+               :stage fluree-node-sdk/stage
+               :status fluree-node-sdk/status}}
```

Comment:
Can you line these values up in the same column? Still need to get some .editorconfig and/or CLJ formatting metadata in-repo w/ automated checks. This stuff is nice to keep consistent but only when automated. Not for this PR, of course... :)